### ISSUE TITLE
posts not downloaded

### ISSUE THREAD

**Issue by veezeeee** [sim0n00ps/OF-DL#482]
![error](https://github.com/user-attachments/assets/7916d445-2338-4f6e-9906-75a1d8bd3e5d)

when trying to download, it brings this error saying the posts have already dowloaded but it only does a couple, 
any clue what the issue might be?

**Comment by randomfapper** [comment]
Chances are it's video files... looks like OF is using some new type of DRM or coding to block them if a creator has it enabled.

**Comment by qorrja** [comment]
I have same issue with v1.7.60
v1.7.68 doesn't even show error messages, finished with full of red '?' and nothing downloaded, pics as well

**Comment by melithine** [comment]
> I have same issue with v1.7.60 v1.7.68 doesn't even show error messages, done with full of red '?' and nothing downloaded, pics as well

That sounds like https://github.com/sim0n00ps/OF-DL/issues/484#issuecomment-2244169343

**Comment by qorrja** [comment]
> > I have same issue with v1.7.60 v1.7.68 doesn't even show error messages, done with full of red '?' and nothing downloaded, pics as well
> 
> That sounds like [#484 (comment)](https://github.com/sim0n00ps/OF-DL/issues/484#issuecomment-2244169343)

Yeah v1.7.68 works perfectly now but no clue with v1.7.60, maybe onlyfans update or something?
Anyway thanks bro

**Comment by melithine** [comment]
The 403 Forbidden is being tracked on #466, closing this one.
